### PR TITLE
Account page adventurer's log

### PR DESCRIPTION
**Pull request by the-code-raccoon**

### CODE REVIEW

File: frontend/src/components/AccountAdventurersLog.tsx [hunk 1]
```diff
@@ -0,0 +1,44 @@
...
+import icons from "../assets";
+import "../styles/AccountAdventurersLog.css";
+
+type Props = {
+    accountData: Account;
+};
+
+const AccountAdventurersLog = ({ accountData }: Props) => {
+    const activities = accountData.activities;
+    console.log(activities);
```
Comment (the-code-raccoon) [inline frontend/src/components/AccountAdventurersLog.tsx]: extra console.log

File: frontend/src/components/AccountAdventurersLog.tsx [hunk 2]
```diff
@@ -0,0 +1,44 @@
...
+
+    return (
+        <div className="account-activities">
+            <div className="account-activities__title">Adventurer's Log</div>
+            <div className="account-activities__list">
+                {activities.map((activity) => {
+                    return (
+                        <div className="account-activities__stat">
```
Comment (the-code-raccoon) [inline frontend/src/components/AccountAdventurersLog.tsx]:
```suggestion
                        <div key={activity.date} className="account-activities__stat">
```
need to add a `key` property, can be anything unique

File: frontend/src/types/Account.tsx [hunk 3]
```diff
@@ -9,7 +9,7 @@ export type AccountSkill = {
 
 export type Account = {
     _id: string;
-    activities: { date: string; details: string; text: string }[];
+    activities: { date: string; details: string; text: string; type: string }[];
```
Comment (the-code-raccoon) [inline frontend/src/types/Account.tsx:12]: create a typescript type for `type` is better than `string` but is okay for now

File: frontend/src/components/AccountAdventurersLog.tsx (hunk 1 above)
Comment (Gahmbit) [inline frontend/src/components/AccountAdventurersLog.tsx]:
```suggestion
```

**Review by Gahmbit — APPROVED** [review]

**Pull request merged**